forrest.go

```diff
@@ -8,6 +22,49 @@ type Forest struct {
 	Intercept float64
 }
 
+type ForestModel struct {
+	Forest      *Forest
+	InBag       [][]float64
+	Predictions [][]float64
+	Importance  *[]*RunningMean
+}
+
+type ForestConfig struct {
```

Comment:
Yeah I decided move the model configuration to a `struct` because I'm not a huge fan of functions with arity of 10+..